src/yadism/coefficient_functions/fonll/kernels.py

```diff
@@ -194,7 +194,7 @@ def generate_heavy_diff(esf, nl, pto_evol):
         asy_weights = heavy.kernels.nc_weights(
             esf.info.coupling_constants, esf.Q2, kind, nl, ihq
         )
-        if kind != "F3":
+        if kind not in ["F3", "gL", "g4"]:
```

Comment:
How about adding a (static) function in obsname module `is_pv` or `is_parity_violationg` or `parity_violation` or similar which return exactly this if condition?

Reply:
Yes, I agree this would be ideal.

Reply:
This is done now! Some functions had to be extended but I think this will not break backward compatibility.